web/app/js/components/MetricsTable.jsx

```diff
@@ -59,6 +59,15 @@ const columnDefinitions = (resource, namespaces, onFilterClick, showNamespaceCol
     }
   ];
 
+  let meshedColumn = {
+    title: formatTitle("M", "Meshed"),
+    dataIndex: "pods",
+    key: "pods",
+    className: "numeric",
+    sorter: (a, b) => numericSort(a.pods.running, b.pods.running),
```

Comment:
I would like the sorting to stay consistent with what we're doing in the ServiceMesh page's Meshed column. So I'd update that sorter to sort on running instead of total.